app/models/authorization_request.rb

```diff
@@ -315,4 +315,11 @@ def applicant_belongs_to_organization
 
     errors.add(:applicant, :belongs_to)
   end
+
+  def interpolated_access_link
```

Comment:
```suggestion
  def access_link
```

Dans le cadre de la demande ça me semble logique comme nom.